packages/eslint-plugin/rules/jsx-a11y-anchor-has-content.js

```diff
@@ -0,0 +1,172 @@
+/**
+ *
+ * @file Wraps eslint-plugin-jsx-a11y/anchor-has-content to allow usage with createInterpolateElement.
```

Comment:
```suggestion
 * @file Wraps around the rule jsx-a11y/anchor-has-content to allow usage with createInterpolateElement.
```

Reply:
would it make sense to add a link to jsx-a11y/anchor-has-content in the file description too? or is this clear enough?